create-doks-with-terraform-flux/README.md

```diff
@@ -1 +1,470 @@
+    cd "$TF_VAR_github_repository_name"
+    ```
+3. Before creating the resources let's store their name in dedicated environment variables in order to fetch them easily later on. Please replace the `<>` placeholders with something meaningful for your case:
+
+    ```bash
+    export FLUX_CD_SOURCE_NAME="<source_component_name>"
+    export FLUX_CD_KUSTOMIZE_NAME="<kustomize_component_name>"
+    ```
+4. Create the git `Source` component and commit the changes:
+
+    ```bash
+    flux create source git "$FLUX_CD_SOURCE_NAME" \
+        --url="https://github.com/${TF_VAR_github_owner}/${TF_VAR_github_repository_name}.git" \
+        --branch="$TF_VAR_github_repository_branch" \
+        --interval=30s \
+        --export > "./${TF_VAR_github_repository_target_path}/${FLUX_CD_SOURCE_NAME}-source.yaml"
+
+    git add -A && git commit -am "Adding the Flux CD Source component"
+    ```
+5. Next we create the path in the git repository where we store our applications deployment yaml files. As an example we will create a simple busybox deployment. Then we build the `Kustomize` resource and commit the changes:
+
+    ```bash
+    APPS_PATH="./apps/busybox"
+
+    mkdir -p $APPS_PATH
+
+    # busybox Namespace yaml file
+    cat << EOF > "${APPS_PATH}/busybox-ns.yaml"
+    apiVersion: v1
+    kind: Namespace
+    metadata:
+        creationTimestamp: null
+        name: busybox
+    spec: {}
+    status: {}
+    EOF
+
+    # busybox Pod yaml file
+    cat << EOF > "${APPS_PATH}/busybox.yaml"
+    ---
+    apiVersion: v1
+    kind: Pod
+    metadata:
+        name: busybox1
+        namespace: busybox
+    labels:
+        app: busybox1
+    spec:
+        containers:
+        - image: busybox
+            command:
+              - sleep
+              - "3600"
+              imagePullPolicy: IfNotPresent
+            name: busybox
+        restartPolicy: Always
+    EOF
+
+    # busybox Kustomization yaml file
+    cat << EOF > "${APPS_PATH}/kustomization.yaml"
+    apiVersion: kustomize.config.k8s.io/v1beta1
+    kind: Kustomization
+
+    commonLabels:
+        app: busybox
+
+    namespace: busybox
+
+    resources:
+       - busybox.yaml
+       - busybox-ns.yaml
+    EOF
+
+    flux create kustomization "$FLUX_CD_KUSTOMIZE_NAME" \
+        --source="$FLUX_CD_SOURCE_NAME" \
+        --path="$APPS_PATH" \
+        --prune=true \
+        --validation=client \
+        --interval=5m \
+        --export > "./${TF_VAR_github_repository_target_path}/${FLUX_CD_KUSTOMIZE_NAME}-kustomization.yaml"
+
+    git add -A && git commit -am "Adding the Flux CD Kustomize component"
+    ```
+6. The final step would be to push the changes:
+
+    ```bash
+    git push origin $TF_VAR_github_repository_branch
+    ```
+
+## Inspecting the results
+
+After a while the busybox namespace and associated pod should be created and running. Let's see what flux has to say about it first:
```

Comment:
```suggestion
After a while, the busybox namespace and associated pod should be created and running. Let's see what Flux has to say about it first:
```

Reply:
Thanks for the suggestion. Fixed in the latest commit.